app/models/order.rb

```diff
@@ -1,19 +1,13 @@
 class Order < ApplicationRecord
   has_many :order_items
   
-  
-  # validates_presence_of :email_address, :mailing_address, :customer_name, :cc_number, :cc_expiration, :cc_security_code, :zip_code, :cart_status
-  # validates :cc_number, length: { is: 16 }  
-  # validates :cc_security_code, length: { is: 3 }  
-  # validates :zip_code, length: { is: 5 }  
-  
+  validates_presence_of :email_address, :mailing_address, :customer_name, :cc_number, :cc_expiration, :cc_security_code, :zip_code, :cart_status, on: :update 
```

Comment:
i did not know validates_presence_of was a thing! will add this to other models too